src/css/components/buttons.css

```
@@ -90,3 +91,11 @@ input[type="submit"]:disabled,
     calc(var(--base-spacing) + 0.1rem);
   font-size: 1rem;
 }
+
+.btn-fluid {
+  width: 100%;
+}
+
+.btn-no-margin {
+  margin: 0;
+}
```

(Retour à la ligne en fin de fichier.)

```suggestion
}
```